docs/gallery/interactive/pyfar_filter_types.ipynb

```diff
@@ -0,0 +1,91 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Overview of filters\n",
+    "\n",
+    "The [pyfar.dsp.filter](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.filter.html) module contains different filter types that are briefly introduced in the following. The filters can be used to directly filter pyfar *Signals* or can return a pyfar *Filter* object. For more information on this refer to the LINK-TO-FILTERING-EXAMPLES` and `LINK-TO-AUDIO-OBJECT-EXAMPLES`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## High-pass, low-pass, band-pass, and band-stop filters\n",
+    "\n",
+    "These are the classic filters that are wrapped from *scipy.signal* and\n",
+    "available from the functions [pyfar.dsp.filter.butterworth](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.filter.html#pyfar.dsp.filter.butterworth), [pyfar.dsp.filter.bessel](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.filter.html#pyfar.dsp.filter.bessel), [pyfar.dsp.filter.chebyshev1](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.filter.html#pyfar.dsp.filter.chebyshev1), [pyfar.dsp.filter.chebyshev2](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.filter.html#pyfar.dsp.filter.chebyshev2), and [pyfar.dsp.filter.elliptic](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.filter.html#pyfar.dsp.filter.elliptic).\n",
+    "\n",
+    "![Standard filter types](../../resources/filter_types_standard.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Linkwitz-Riley cross-over\n",
+    "\n",
+    "The function [`pyfar.dsp.filter.crossover`](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.filter.html#pyfar.dsp.filter.crossover) implements Linkwitz-Riley cross-over filters that are often used in loudspeaker design. The magnitude of the filters at the cross-over frequency is -6 dB and because the filters are in phase, their output adds to a constant magnitude response.\n",
```

Comment:
```suggestion
    "The function [pyfar.dsp.filter.crossover](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.filter.html#pyfar.dsp.filter.crossover) implements Linkwitz-Riley cross-over filters that are often used in loudspeaker design. The magnitude of the filters at the cross-over frequency is -6 dB and because the filters are in phase, their output adds to a constant magnitude response.\n",
```

Reply:
Ups, done